Blueprints/BlueprintDefinitions/vs2022/AnnotationsFramework/template/.template.config/template.json

```diff
@@ -5,7 +5,7 @@
     "Lambda",
     "Serverless"
   ],
-  "name": "Lambda Annotations Framework (Preview)",
+  "name": "Lambda Annotations Framework",
```

Comment:
Should we add the ` Sample` suffix to match what we did for the VS manifest?

Reply:
Added